notes/zrq/20210206-02-echo-S3.txt

```diff
@@ -0,0 +1,44 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2021, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#zrq-notes-time
+#zrq-notes-indent
+#zrq-notes-crypto
+#zrq-notes-ansible
+#zrq-notes-osformat
+#zrq-notes-zeppelin
+#
+
+    Target:
+
+        Test out the AWS S3 libraries in Hadoop with data in an Echo S3 account.
+
+    Result:
+
+        Work in progress ....
+
+
+
+    #
+    # Use the command line client to push data into our S3 account.
+    # Use the Hadoop libraries to query the data ...
+    #
+
+
```

Comment:
Was there supposed to be some more documentation in this file that is missing?

Reply:
This is as far as I got on that day, just notes to self on what we needed to do. 
Not much I know - I planned to do more but got diverted ..